zerver/tests/test_bugdown.py

```diff
@@ -728,6 +728,33 @@ def test_mit_rendering(self):
             '<p><a href="https://lists.debian.org/debian-ctte/2014/02/msg00173.html" target="_blank" title="https://lists.debian.org/debian-ctte/2014/02/msg00173.html">https://lists.debian.org/debian-ctte/2014/02/msg00173.html</a></p>',
             )
 
+    def test_render_italic(self):
+        # type: () -> None
+        msg = "*foo*"
+        converted = bugdown.convert(msg)
+        self.assertEqual(
+            converted,
+            "<p><i>foo</i></p>")
+
+        msg = "*foo bar*"
+        converted = bugdown.convert(msg)
+        self.assertEqual(
+            converted,
+            "<p><i>foo bar</i></p>")
+
+        msg = "const char *x = (char *)y"
+        converted = bugdown.convert(msg)
+        self.assertEqual(
+            converted,
+            "<p>const char *x = (char *)y</p>")
+
+        msg = "* item1\n* item2"
+        converted = bugdown.convert(msg)
+        self.assertEqual(
+            converted,
+            "<ul>\n<li>item1</li>\n<li>item2</li>\n</ul>")
```

Comment:
Why not just put these tests in `bugdown-data.json` so we can test with both markdown processors?